epoxy-adapter/src/main/java/com/airbnb/epoxy/BaseEpoxyAdapter.java

```diff
@@ -140,6 +138,32 @@ protected void onModelBound(EpoxyViewHolder holder, EpoxyModel<?> model, int pos
 
   }
 
+  @CallSuper
+  @Override
+  public void onAttachedToRecyclerView(RecyclerView recycler) {
+    super.onAttachedToRecyclerView(recycler);
+
+    // set our item animator decorator so we can intercept calls to the item animator
+    // and the user doesn't have to worry about setting the decorator himself
+    if (!(recycler.getItemAnimator() instanceof ChangeAnimationDelegator)) {
```

Comment:
If their custom animator is for insertions/removals and they don't want to do anything for item changes, then would we still run that reflection code in the delegate animator? Just wondering if they might not want to use our delegate for performance reasons if they don't need the item change animation support. Regardless, it seems like there should be _some_ way to override our wrapper if they have some unforseen case. I don't want to force it on users if they don't want it.

I would imagine that most people only want the item change animations so they wouldn't have their own animator that needs wrapping (at least that's the case for our team), so the automatic animator handling would still be awesome for those cases.